tests/hazmat/backends/test_openssl.py

```diff
@@ -169,6 +169,32 @@ def test_bn_to_int(self):
         bn = backend._int_to_bn(0)
         assert backend._bn_to_int(bn) == 0
 
+    def test_bn_rand(self):
```

Comment:
We don't need tests for bindings like these. This code can be removed.

Reply:
👍